client/src/reader/components/section/body-nodes/nodes/Math/Node.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import * as ReactDOMServer from "react-dom/server";
+import PropTypes from "prop-types";
+import {
+  getAnnotationStyles,
+  getlocalAnnotationsArray
+} from "./annotationHelpers";
+import { useTranslation } from "react-i18next";
+import { uid } from "react-uid";
+import { useErrorHandler } from "react-error-boundary";
+
+const createNode = n =>
+  React.createElement(
+    n.tag,
+    {
+      ...n.attributes,
+      key: n.nodeUuid ?? uid(n),
+      "data-node-uuid": n.nodeUuid,
+      "data-text-digest": n.textDigest
+    },
+    n.content ? n.content : n.children?.map(child => createNode(child))
+  );
+
+function MathNode({
+  attributes,
+  children,
+  openAnnotations,
+  uuids,
+  hasInteractiveAncestor
+}) {
+  const { t } = useTranslation();
+  const handleError = useErrorHandler();
+
+  const childNodes = () => {
+    try {
+      return children.map(child => createNode(child));
+    } catch (e) {
+      handleError();
+    }
+  };
+
+  const localAnnotations = getlocalAnnotationsArray(openAnnotations);
+  const {
+    classes,
+    removableHighlightId,
+    textAnnotationIds,
+    annotationIds,
+    interactiveAttributes
+  } = getAnnotationStyles(localAnnotations, uuids, t, hasInteractiveAncestor);
+
+  const Wrapper = attributes.display === "inline" ? "span" : "div";
+  const wrapperStyles = Wrapper === "div" ? { width: "max-content" } : {};
```

Comment:
You may want to add `{ maxWidth: "100%" }`. `width: max-content` will cause an element to ignore the width of its parent and extend right off the page if it's really that wide.